testsrc/org/mozilla/javascript/tests/es6/NativeArray2Test.java

```diff
@@ -0,0 +1,68 @@
+package org.mozilla.javascript.tests.es6;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.mozilla.javascript.Context;
+import org.mozilla.javascript.ScriptableObject;
+
+/**
+ * @see <a
+ *     href="https://github.com/mozilla/rhino/issues/673">https://github.com/mozilla/rhino/issues/673</a>
```

Comment:
Is this comment correct? It looks to me like the issue not directly related to this Pull Request.

Reply:
give me a second...